extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -198,7 +198,7 @@ public function save() {
 		$this->response->setVal( 'status', $status );
 
 		if ( $status ) {
-			wfRunHooks( 'CuratedContentSave' );
+			wfRunHooks( 'CuratedContentSave', [ $sections ] );
```

Comment:
kk, just saying that I could use that param instead of rebuilding it in the function. I (or someone else who will develop that) could think of that.